pkg/apps/utils.tsx

```diff
@@ -49,7 +56,11 @@ export function icon_url(path_or_url) {
     return prefix + '?' + query;
 }
 
-export const ProgressBar = ({ size, title, data, ariaLabelledBy }) => {
+export const ProgressBar = ({ size, data, ariaLabelledBy }: {
+    size?: "lg" | "md" | "sm",
+    data: { percentage: number, waiting: boolean },
```

Comment:
I guess `data` is actually the same object that is also passed to `CancelButton`. Could we make a real type (or interface) for it?

Reply:
`interface JobData { percentage: number, waiting: boolean, cancel: ... }`. I haven't checked the callers yet....